CODEOWNERS

```diff
@@ -4,33 +4,7 @@
 #
 # See https://help.github.com/articles/about-codeowners/
 
-* @bajtos @raymondfeng
-
-packages/authentication/* @bajtos @raymondfeng
-packages/boot/* @raymondfeng @hacksparrow
-packages/booter-lb3app/* @bajtos @nabdelgadir
-packages/build/* @bajtos @raymondfeng
-packages/http-caching-proxy/* @bajtos
-packages/http-server/* @hacksparrow @bajtos
-packages/cli/* @raymondfeng @bajtos
-packages/context/* @bajtos @raymondfeng
-packages/core/* @bajtos @raymondfeng
-packages/eslint-config/* @raymondfeng
-packages/metadata/* @raymondfeng
-packages/openapi-spec-builder/* @bajtos @raymondfeng
-packages/openapi-v3/* @bajtos @jannyHou
-packages/openapi-v3-types/* @bajtos @jannyHou
-packages/repository/* @raymondfeng
-packages/repository-json-schema/* @bajtos
-packages/rest/* @bajtos @raymondfeng
-packages/service-proxy/* @raymondfeng
-packages/testlab/* @bajtos
-examples/todo/* @bajtos @hacksparrow
-examples/express-composition/* @nabdelgadir
-examples/greeter-extension/* @raymondfeng
-examples/hello-world/* @b-admike
-examples/lb3-application/* @bajtos @nabdelgadir
-examples/log-extension/* @hacksparrow
-examples/rpc-server/* @hacksparrow
-examples/soap-calculator/* @marioestradarosa
-docs/* @bajtos @raymondfeng @hacksparrow @b-admike @jannyHou
+# The lab feature owners can watch the code changes
+# in the lab branch `lab/<my-feature-name>` by adding
+# their names like:
+# lab/passport-adapter/* @jannyHou
```

Comment:
👍

Reply:
`lab/<pkg_name>` -> `labs/<pkg_name>`?

Reply:
@jannyHou ☝️ I think we still want this changed to `labs/` since the experimental features will live there

Reply:
yep updated =p